src/build-query.js

```diff
@@ -0,0 +1,70 @@
+const _ = require('lodash');
+const { language } = require('../config');
+
+const buildQuery = (config, withAcl) => {
```

Comment:
What is the reason to pass `withAcl` via method argument but fetch `language` through the config? I would expect this ...

```
const { language, withAcl } = require('../config');

const buildQuery = (config) => {
``` 

or this 

```
const buildQuery = (config, withAcl, language) => {
```

But that can stay that way. I will remove the "language" config parameter completely today / tomorrow because it's not needed anymore. There was a time where the default language from this library (english) was different to the default language of the api (german). So we had to provide the default language param for every call. But the api changed in the meantime and return english results by default now.

Reply:
ideally I wanted to pass withAcl through the config as well, but being able to get a clean test was proving tricky without further refactoring, it's definitely something I will continue to look at and perhaps provide a good solution, or else you or another contributor may see an elegant solution as well. Either way your repo now has good behaviour unit tests for your CI, it might be nice to get a smallish e2e test and a linter to really setup up the repo well for future contributions.

Reply:
You are right. I will create some new issues later ;-)